### DISCUSSION TITLE
webmind und sae laufen nicht an nach der Installation

### DISCUSSION THREAD

**Discussion by ChLSchulz** [camueller/SmartApplianceEnabler#597] [Installation und Betrieb]
Guten Abend, habe gerade zweimal versucht, die manuelle Installation  nach Vorgabe durchzuführen - leider ohne Erfolg, weder Webmin noch sae laufen an (keine Antwort auf port 8080 und port 10000). Was kann ich tun?
[install.log](https://github.com/camueller/SmartApplianceEnabler/files/14827750/install.log)

Host-Name ist raspi, die einzige Veränderung, die ich vorgenommen habe: der Benutzername ist nicht pi sondern cs.

**Comment by camueller** [comment]
Zumindest während der Installation bestanden Netzwerkprobleme:
```
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8002::154|:443... failed: Network is unreachable.
```

**Comment by ChLSchulz** [comment]
Habe heute die Installation nochmal laufen mit LAN-Anschluss laufen lassen, gleiches Ergebnis. Auf dem Bildschirm werden Fehler eingeblendet, diese sind im install.log nicht zu sehen. Dateien anbei...
[install.log](https://github.com/camueller/SmartApplianceEnabler/files/14841696/install.log)
[SAE Install.screenlog.txt](https://github.com/camueller/SmartApplianceEnabler/files/14841697/SAE.Install.screenlog.txt)
 (Install.Screenlog) ist die Mitschrift der Installation...

**Comment by camueller** [comment]
Wie beim ersten Versuch - Netzwerkprobleme gleich zu Beginn - da kann ich nichts machen, das liegt an Deiner Infrastruktur.
Sieht so aus, als ob Du IP v6 verwendest ...
```
root@raspi:/home/pi# curl -sSL https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/setup.sh | sh
--2024-04-02 18:28:01--  https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/install2.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 2606:50c0:8000::154, 2606:50c0:8001::154, 2606:50c0:8002::154, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8000::154|:443... failed: Network is unreachable.
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8001::154|:443... failed: Network is unreachable.
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8002::154|:443... failed: Network is unreachable.
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8003::154|:443... failed: Network is unreachable.
```

Übrigens machst Du nicht die manuelle Installation, sondern die "Installation via Script".

Probier doch mal die manuelle Installation: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/InstallationManual_DE.md

**Comment by ChLSchulz** [comment]
Sorry, war einige Tage unterwegs und kam erst heute dazu, die manuelle Installation durchzuprobieren. Die scheitert bei der Installation von java mit der Fehlermeldung "ca-certificates-java : Breaks: openjdk-11-jre-headless (< 11.0.19+7~1~) but 11.0.17+8-2 is to be installed". Rest (expand-rootfs, apt update, apt upgrade) lief durch.

[SAE Install.screenlog.txt](https://github.com/camueller/SmartApplianceEnabler/files/14897099/SAE.Install.screenlog.txt)

Die bei google findbaren Lösungen für den Zertifikatfehler habe ich ohne Erfolg probiert. Jemand eine Idee für eine Lösung?

**Comment by camueller** [comment]
Dieser Fehler hat ja nichts mit dem SAE zu tun sondern mit Deinem System. Deshalb kannst Du auch jeder Anleitung oder Hinweis nachgehen, die Dich dahin führt, dass Du eine Java11-Runtime auf Deinem System installiert bekommst.
